.gitattributes

```diff
@@ -1,6 +1,8 @@
 # Set the default behavior to LF, because checkstyle enforces it
 * text eol=lf
 
+*.bat eol=crlf
```

Comment:
No idea why these changes appear here but they should not. They are already on master following the Gradle 2.14 upgrade